src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,131 +1,146 @@
-      //A for shooting (digital input limit switch)
-      _operator 
-      .a()
-      .whileTrue(new RunCommand(_trapRollers::TrapRollersIn, _trapRollers));
-  }
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-  */ 
-  public Command getAutonomousCommand() {
-    // An example command will be run in autonomous
-    return _chooser.getSelected();
-  }
-}
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+//import frc.robot.commands.Autos;
+import frc.robot.subsystems.*;
+import frc.robot.commands.*;
+
+import edu.wpi.first.networktables.DoubleTopic;
+import edu.wpi.first.networktables.IntegerTopic;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+//import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.commands.*;
+import frc.robot.subsystems.*;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and trigger mappings) should be declared here.
+ */
+public class RobotContainer {
+  // The robot's subsystems and commands are defined here...
+  public final Drive_Train _drive_Train = new Drive_Train();
+  //private final Joystick _driver = new Joystick(0);
+  //private final Joystick _operator = new Joystick(1);
+  public final CommandXboxController _driver = new CommandXboxController(0);
+  private final CommandXboxController _operator = new CommandXboxController(1);
+
+  private final LeftClaw _leftClaw = new LeftClaw();
+  private final Claw _rightClaw = new RightClaw();
+  private final Intake_Bar _intakeBar = new Intake_Bar();
+  private final Intake_Belt _intakeBelt = new Intake_Belt(){};
+  private final Trap_Rollers _trapRollers = new Trap_Rollers();
+  private final Limit_Switch _limitSwitch = new Limit_Switch(9);
+  private DoubleTopic centerTagX;
+  private IntegerTopic centerImageX;
+  private SendableChooser<Command> autoChooser = new SendableChooser<Command>();
+  private SendableChooser<String> allianceChooser = new SendableChooser<String>();

```

Comment:
I think we can get rid of this

Reply:
allianceChooser removed (not the other stuff though, that needs to stay)